src/main/resources/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder/config.jelly

```diff
@@ -156,7 +157,43 @@
 
             }
 
-            switcherLastConfiguration_${descriptor.defaultBuildID()} = function(){
+            switchTestInfoInput_${descriptor.defaultBuildID()} = function (container) {
+            	//current selected input type, from the select HTML element
+            	var selectTestInfoInputType = Q("#selectTestInfoInputType_${descriptor.defaultBuildID()}").find('option:selected').val();
+
+            	if (!selectTestInfoInputType)
```

Comment:
1) missing curly brackets
2) If we get inside this if, shouldn't we write an console.error() with the problem?